CONTRIBUTING.md

```diff
@@ -11,8 +11,30 @@ Contributing to Zeppelin (Source code, Documents, Image, Website) means you agre
 ## SourceControl Workflow
 Zeppelin follows [Fork & Pull] (https://github.com/sevntu-checkstyle/sevntu.checkstyle/wiki/Development-workflow-with-Git:-Fork,-Branching,-Commits,-and-Pull-Request) model.
 
+
+## The Review Process
+
+When a Pull Request is submitted, it is being merged or rejected by following review process.
+
+* Anybody can be a reviewer and may comment on the change and suggest modifications.
+* Reviewer can indicate that a patch looks suitable for merging with a comment such as : "Looks good", "LGTM", "+1".
+* Pullrequest is open for 1 or 2 days for potential additional review, unless it's got enough indication of suitable for merging.
```

Comment:
I think this step should be below, since its after a committer gave an indication of suitable for merging